tests/store/test_store_io.h

```diff
@@ -162,6 +162,167 @@ SEQAN_DEFINE_TEST(test_store_io_read_ucsc_known_genes)
     SEQAN_ASSERT(atEnd(it));
 }
 
+SEQAN_DEFINE_TEST(test_store_io_read_ucsc_known_genes_and_isoforms)
+{
+    // The file contains 13 annotations in total which will be checked line
+    // after line.
+    seqan::CharString ucscGenesPath = SEQAN_PATH_TO_ROOT();
+    append(ucscGenesPath, "/tests/store/example_knownGene.txt");
```

Comment:
please use ```getAbsolutePath``` instead of SEQAN_PATH_TO_ROOT().
See here the documentation: http://docs.seqan.de/seqan/develop/